src/server/client.ts

```diff
@@ -69,8 +68,10 @@ namespace ts.server {
         }
 
         private processRequest<T extends protocol.Request>(command: string, args?: any): T {
-            var request: protocol.Request = {
-                seq: this.sequence++,
+            const seq = this.sequence;
+            this.sequence++;
+            const request: protocol.Request = {
+                seq,

```

Comment:
it would a little terser to say `const request = { this.sequence, ... }; this.sequence++;` instead of saving to a local.